src/main/scala/com/workday/warp/persistence/mysql/WarpSlickFunctions.scala

```diff
@@ -0,0 +1,26 @@
+package com.workday.warp.persistence.mysql
+
+import com.workday.warp.persistence.mysql.WarpSlickFunctions._
+import slick.ast.Library.SqlAggregateFunction
+import slick.ast.TypedType
+import slick.lifted.FunctionSymbolExtensionMethods._
+import slick.lifted.{Query, Rep}
+import scala.language.{higherKinds, implicitConversions}
+
+/**
+  * Created by ruiqi.wang
+  *
+  */
+protected class WarpSlickFunctions[B1, P1, C[_]](val q: Query[Rep[P1], _, C]) extends AnyVal {
```

Comment:
why does this extend `AnyVal`?

Reply:
Hmm, I was basing off the slick source code here: https://github.com/slick/slick/blob/master/slick/src/main/scala/slick/lifted/ExtensionMethods.scala#L156-L173

I removed the extension and the tests still work so 🤷‍♂